seedfarmer/__init__.py

```diff
@@ -41,7 +62,19 @@
 
 
 def enable_debug(format: str) -> None:
```

Comment:
nit - `format` looks to be an unused parameter now

Reply:
hmmm...that should not have gotten through the format / validate steps...will dig in